### PR TITLE
Suppressing rescaling of the learning rate (#12)

### PR DESCRIPTION
**Pull request by marylou-gabrie**
Suppressing rescaling of the learning rate by the number of visible units in fit_batch!

### CODE REVIEW

**Comment by dfdx** [comment]
This rescaling came from original code in SciKit Learn, and I never had a chance to think it out. If you can confirm that results for, say, MNIST example are not deteriorated, I will gladly merge this PR.

**Comment by marylou-gabrie** [comment]
In [SciKit](https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/neural_network/rbm.py) learn, the learning gets rescaled by the number of training samples, i.e. the size of the minibatch (see line 274, where v_pos dimensions are explained in line 264). This is an understandable rescaling convention that is commonly used, for example in SciKit learn or in G. Hinton [demo code](http://www.cs.toronto.edu/~hinton/code/rbm.m). I updated my pull request accordingly.

**Comment by dfdx** [comment]
Ah, sorry, I misread your comment. Yes, this is a bug, thanks for fixing it.
